### ISSUE TITLE
[Feat]: More controls need Tab Index

### ISSUE THREAD

**Issue by ScottKemp** [lowcoder-org/lowcoder#1883]
**Is your feature request related to a problem? Please describe.**
When trying to quickly enter data into a form as an end user, I click tab to move from one field to another.  The tab order is all over the place.  I kind of expected it to just work as is, top to bottom.

I did see the Tab Index in the Edit Input control, and that does work.  But then there is no Tab Index property for the Select Control or Button, for example.  I suspect many other controls are also missing it.  Without this or some kind of predictable auto tab order, tabbing is essentially broken.

**Describe the solution you'd like**
Either get an auto tab solution to work, Or Add Tab Index to all controls.

**Describe alternatives you've considered**
No alternatives

**Additional context**

**Comment by adnanqaops** [comment]
@iamfaran , Kindly, add Tab index property to following components as well : 

1. File Upload
2. Scanner 
3. Multi Tags
4. Cascader
5. Float Button
6. Tree Select
7. Rating

**Comment by adnanqaops** [comment]
Tab Index control has now been added to mentioned components on Cloud ( **V2.7.4** )
